apps/site/sentry.client.config.ts

```diff
@@ -1,14 +1,30 @@
 // Context: https://docs.sentry.io/platforms/javascript/guides/nextjs/
 
 import * as Sentry from "@sentry/nextjs";
-import type { NextjsOptions } from "@sentry/nextjs/types/utils/nextjsOptions";
+import { Replay } from "@sentry/replay";
 
 const dsn = process.env.NEXT_PUBLIC_SENTRY_DSN;
 
-export const sentryConfig: NextjsOptions = {
+const replaysSamplingRate = Number.parseInt(
```

Comment:
@kachkaev Do we want to update this to `parseFloat`, too, as flagged in https://github.com/hashintel/hash/pull/1232#discussion_r1001141982?

Reply:
This is being addressed in https://github.com/blockprotocol/blockprotocol/pull/698